app/microservice_tables.py

```diff
@@ -240,7 +322,13 @@ def get_table_records(field_classes, json_records, remove_html=False):
     combined_field = []
     for field in field_classes:
       if field.show_as_table_header:
-        record_field_data = each_record[field.database_name]
+        if field.name:
```

Comment:
Minor style note, but generally, we don't want to put HTML into the Python code - it should be in a template, or sub-template. For now, this is okay.

Reply:
👍